### PR TITLE
feat(examples,docs): add Sidra example

### PR DESCRIPTION
**Pull request by barbarbar338**
This PR adds [Sidra](https://npmjs.com/sidra) example.

### CODE REVIEW

File: examples/sidra/package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (QuiiBz) [inline examples/sidra/package.json:1]: Please follow the structure with `name`, `version`, `private`, & `lagon-dev` script

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks! Could you also run `pnpm changeset` at the root of the repo and select the sidra example + the docs to add this change to their respective changelog?

**Comment by barbarbar338** [comment]
I guess this is okay rn :)

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks!

**Comment by QuiiBz** [comment]
I'm reverting https://github.com/lagonapp/lagon/pull/914/commits/5f8c801e0fe590b94e56fc6b5c619f7e945902e3, the version of PNPM hasn't been updated.

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**